From adc13c983d4a53b632d16bafb47f0c052a5403be Mon Sep 17 00:00:00 2001
From: Daniel Golle <daniel@makrotopia.org>
Date: Sat, 3 Jun 2017 20:00:03 +0200
Subject: [PATCH] MIPS: pci-mt7620: enabled PCIe on MT7688

Use PCIe support for MT7628AN also on MT7688.
Tested on WRTNODE2R.

Signed-off-by: Daniel Golle <daniel@makrotopia.org>
---
 arch/mips/pci/pci-mt7620.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/mips/pci/pci-mt7620.c b/arch/mips/pci/pci-mt7620.c
index 27ac00c36bc0..d67de97ef34e 100644
--- a/arch/mips/pci/pci-mt7620.c
+++ b/arch/mips/pci/pci-mt7620.c
@@ -316,6 +316,7 @@ static int mt7620_pci_probe(struct platform_device *pdev)
 		break;
 
 	case MT762X_SOC_MT7628AN:
+	case MT762X_SOC_MT7688:
 		if (mt7628_pci_hw_init(pdev))
 			return -1;
 		break;
-- 
2.11.0

